README_user.md

```diff
@@ -486,6 +486,99 @@ Diffing SERIAL/wrfout_d01_2000-01-24_12:00:00 wrfout_d01_2000-01-24_12:00:00
 
 ### Checking WRF DA results
 
+1. Build the WRFDA code
+
+WRFDA can be built in 4DVar mode or non-4DVar mode. The 4DVar build allows a user to also run
+3DVar and hybrid-3D/4DEnVar. The 4DVar build needs to additionally build WRFPlus (i.e., the tangent linear and adjoint (TL/AD) of WRF).
+Users may find a benefit to having separate directories for WRF, WRFPLUS, and WRFDA.
+
+For WRFPlus build:
+```
+cd ~
+cp -pr WRF WRFPLUS
+cd WRFPLUS 
+./configure wrfplus << EOF
+18
+EOF
+./compile wrfplus >& foo
+ls -ls main/*.exe
+-rwxr-xr-x 1 liuz ncar 64177872 Mar 12 10:56 wrfplus.exe
+```
+
+For WRFDA-4DVar build:
+```
+cd ~
+cp -pr WRF WRFDA
+cd WRFDA
+setenv CRTM 1   # will build with CRTM, optinal
+setenv RTTOV rttov-lib-directory # 3rd part software, optional
```

Comment:
@liujake 
Jake,
This is not in the current container. Do we tell users how to get this (where to put it, how to build it) or just not include it at all?

Reply:
CRTM is inside WRFDA package. RTTOV is optional and can be skipped for now. I can add more info about where to get RTTOV in the future when we include WRFDA test cases.

Reply:
@davegill rttov line (and hdf5 line) can be removed if you like.